MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -287,7 +288,10 @@ - (void)userNotificationCenter:(NSUserNotificationCenter *)center
 }
 
 - (void)forwardInvocation:(NSInvocation *)anInvocation {
-  if ([self.originalUserNotificationCenterDelegate respondsToSelector:[anInvocation selector]]) {
+
+  // Testing if the selector is defined in NSUserNotificationCenterDelegate or not.
+  struct objc_method_description hasMethod = protocol_getMethodDescription(@protocol(NSUserNotificationCenterDelegate), [anInvocation selector], NO, YES);
+  if (hasMethod.name != nil && [self.originalUserNotificationCenterDelegate respondsToSelector:[anInvocation selector]]) {
```

Comment:
I would just use `NULL` instead of `nil` since we are not comparing an objc object: `hasMethod.name != NULL`.

Reply:
Done.